gulpfile.js

```diff
@@ -3,14 +3,17 @@ var path = require('path'),
     uglify = require('gulp-uglify'),
     concat = require('gulp-concat'),
     watch = require('gulp-watch'),
-    rimraf = require('gulp-rimraf'),
+    del = require('del'),
```

Comment:
I had just noticed the deprecation in the rimraf docs, and here you're already on it